cmd/node_builder.go

```diff
@@ -128,3 +132,24 @@ type NodeConfig struct {
 	RootSeal    *flow.Seal
 	RootChainID flow.ChainID
 }
+
+func DefaultBaseConfig() *BaseConfig {
```

Comment:
Similar to the `DefaultAccessNodeConfig`, a struct to hold the default `BaseConfig` values.